### PR TITLE
add default loginhelp page

### PR DESCRIPTION
**Pull request by milescalabresi**
There are a few spots (such as "need help?" or "login help" link under the log / sign up buttons) that links to this page /myesp/loginhelp.html, but that page does not exist by default. This creates a default page that admins can modify.

### CODE REVIEW

File: esp/templates/users/loginhelp.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+- If you've forgotten my username/password, you can use our
+[password reset page][2] as long as you still have access to the email you
+used to register.
+The reset will remind you of your username and ask you to reset your password.
+If you do not have access to that email account anymore, [send us an email][3].
+
+- If you tried logging in and it says my password is invalid (even after
```
Comment (hwatheod) [inline esp/templates/users/loginhelp.html]: "my password" -> "your password" here?

File: esp/templates/users/loginhelp.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+password recovery), then chances our your browser doesn't support [cookies][1].
+Please visit [Google's guide to cookies][1].
+
+- If you've gone to the password reset page but you haven't received the rese
```
Comment (hwatheod) [inline esp/templates/users/loginhelp.html]: one more typo: "rese" -> "reset"

File: esp/templates/users/loginhelp.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[password reset page][2] as long as you still have access to the email you
+used to register.
+The reset will remind you of your username and ask you to reset your password.
+If you do not have access to that email account anymore, [send us an email][3].
+
+- If you tried logging in and it says your password is invalid (even after
+password recovery), then chances our your browser doesn't support [cookies][1].
+Please visit [Google's guide to cookies][1].
+
+- If you've gone to the password reset page but you haven't received the rese
```
Comment (hwatheod) [inline esp/templates/users/loginhelp.html]: "rese" -> "reset"

Comment (milescalabresi) [reply esp/templates/users/loginhelp.html]: thanks

**Pull request merged**